spec/fixtures/vcr_cassettes/client/envelopes/cancel/success.yml

```diff
@@ -0,0 +1,444 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://api.signable.co.uk/v1/templates?limit=3&offset=1
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+      Accept:
+      - "*/*"
+      User-Agent:
+      - Ruby
+      Authorization:
+      - Bearer <OAUTH_TOKEN>
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Date:
+      - Wed, 02 Jun 2021 15:28:39 GMT
+      Content-Type:
+      - application/json; charset=UTF-8
+      Transfer-Encoding:
+      - chunked
+      Connection:
+      - keep-alive
+      Server:
+      - nginx
+      Vary:
+      - Accept-Encoding
+      Access-Control-Allow-Origin:
+      - "*"
+      Access-Control-Allow-Methods:
+      - GET,POST,DELETE,PUT,OPTIONS
+      Access-Control-Allow-Headers:
+      - Authorization
+    body:
+      encoding: ASCII-8BIT
+      string: |-
+        {
+            "http": 200,
+            "offset": 1,
+            "limit": 3,
+            "total_templates": "29",
+            "prev": "https://api.signable.co.uk/v1/templates?offset=0&limit=3",
+            "next": "https://api.signable.co.uk/v1/templates?offset=4&limit=3",
+            "templates": [
+                {
+                    "template_id": "16208133",
+                    "template_fingerprint": "0713b744a4f39c46d3d434f195258896",
+                    "template_title": "PARTICIPATION AGREEMENT-TWO-SIGNATORIES",
+                    "template_page_total": "3",
+                    "template_in_progress": "118",
+                    "template_parties_total": "2",
+                    "template_widget_url": "",
+                    "template_widget_embed": "",
+                    "template_pdf_url": "https://docs.signable.co.uk/original/0713b744a4f39c46d3d434f195258896.pdf",
```

Comment:
This link allows me to open the actual participation agreement. It's Smart specific so is it possible to also replace it? (Or maybe use an empty Signable document?

Reply:
Good point.  I'm going to try to filter this out like we do with the API keys...

Reply:
well we did that before, used an empty template. @bugthing  suggested since it's an open source gem we shouldn't use any smart specific id-s. So if you were to pick it up without smart access you could still develop it. but I agree it's not the best solution.

Reply:
👍

Reply:
@RoanMertens we can't filter out ID/fingerprint data from these response bodies because then the test suite won't work, as we assert on a returned envelope fingerprint in one test while multiple tests use the returned template fingerprint and party ID to create and delete envelopes.

We think the solution is to create a new Signable test account used only for running the gem's test suite, containing only one dummy Template document with no Smart branding, company personnel names or any other sensitive information inside it.

We're going to discuss this with Ben when he's back tomorrow.

Reply:
That sounds good to me. Thanks for the update. 🙂

Reply:
I guess we have to wait with merging this pr until after the tests have been run with the dummy template.

Reply:
@RoanMertens  if you check the VCR cassettes, I believe you will no longer find any sensitive info in the linked documents 👍